documentation/en/source/pages/templates-spec/stack-bundles.rst

```diff
@@ -205,10 +205,20 @@ To get the list of categories available, run command:
 
 	$ hammr bundle categories
 
-Adding oses
-~~~~~~~~~~~
+Adding restriction rule
+~~~~~~~~~~~~~~~~~~~~~~~
 
-The following example show how you can provide oses for which the bundle is designed.
+The following example show how you can provide restrictions.
+
+The restriction rule is represented by a "logical" expression.
```

Comment:
The format of the expression is: ``Object#field=value`` or ``Object#field!=value``.
[[Make sure to use `` instead of " to make sure correct format is applied]]